README.md

```diff
@@ -25,112 +25,122 @@ sequenceDiagram
-                end
-                TMS ->> TMS: rebuildCache
-                TMS ->> Cache: setJson(DataCache)
-            end
             par save in ArangoDB
-                TMS ->> Ara: saveTransactionHistory(pacs008)
                 TMS ->> Ara: addAccount(debtor)
                 TMS ->> Ara: addAccount(creditor)
+                TMS ->> Cache: save Data Cache
+                %% TMS ->> Ara: saveTransactionHistory(pacs008)
             end
-            TMS ->> ED: NATS handleResponse: Pacs008 Message
-        end
+            alt Quoting Enabled
+                par save in ArangoDB
+                  TMS ->> Ara: addEntity(creditor)
+                  TMS ->> Ara: addEntity(debtor)
+                end
+
+                par save in ArangoDB
+                  TMS ->> Ara: addAccountHolder(creditor)
+                  TMS ->> Ara: addAccountHolder(debtor)
+                end
+            end
+            TMS ->> Ara: saveTransactionRelationship
+            TMS ->> Ara: saveTransactionHistory(pacs008)
+
+            
         alt Error
         TMS ->> log: Logging of error
         TMS ->> TMS: Throw error
         end
+        TMS ->> ED: NATS handleResponse: Pacs008 Message
         TMS ->> log: Transaction sent to ED service
         TMS ->> Client: /v1/evaluate/iso20022/pacs.008.001.10 POST Result
-    end
+
+    %% ISO20022 Message Pacs002
+        Client ->>+ TMS: POST /v1/evaluate/iso20022/pacs.002.001.12: Pacs002.001.12 Message
+        TMS ->> TMS: Validate Pacs002
+        TMS ->> log: Logging of receipt
+        alt Validation Error
+        TMS ->> log: Logging of error
+        TMS ->> Client: /v1/evaluate/iso20022/pacs.002.001.12 POST Result
+        end
+        TMS ->> Cache: getCache
+        alt cache miss
+          TMS ->> Ara: get Pacs008
+          TMS ->> TMS: rebuild cache
+        end
+        TMS ->> Ara: saveTransactionHistory
+        TMS ->> Ara: get Pacs008
+        TMS ->> Ara: saveTransactionRelationship
+        alt Error
+        TMS ->> log: Logging of error
+        TMS ->> TMS: Throw error
+        end
+        TMS ->> ED: NATS handleResponse: Pacs002 Message
+        TMS ->> log: Transaction sent to ED service
+        TMS ->> Client: /v1/evaluate/iso20022/pacs.002.001.12 POST Result
 ```
 
-![](./images/image-20230815-100606.png)
 
 Transaction monitoring is one of many risk management activities that an organization must perform to ensure healthy operations and also to remain compliant with relevant legislation or regulations.
 
-Currently, it accepts ISO20022 Pain001.001.11 message , pain.013 message , pacs.002.001.12 and pacs.008.001.10. For more information click [here](https://github.com/frmscoe/docs/blob/dev/Knowledge-Articles/iso20022-and-tazama.md)
+Currently, it accepts ISO20022 Pain001.001.11 message, pain.013 message, pacs.002.001.12 and pacs.008.001.10. For more information click [here](https://github.com/frmscoe/docs/blob/dev/Knowledge-Articles/iso20022-and-tazama.md)
```

Comment:
Somewhat inconsistent naming of the different ISO20022 messages here. Recommend:

"Currently, the TMS API is able to accept ISO20022 pain.001.001.11, pain.013.001.09, pacs.002.001.12 and pacs.008.001.10 messages. Depending on the configuration of the TMS API service via the `QUOTING` environment variable, the pain.001 and pain.013 messages may be excluded (`QUOTING=FALSE`)."